### PR TITLE
show all bounty on homepage

### PR DESCRIPTION
**Pull request by Marvel-Ib**
## Describe your changes
Enable the load more button on the homepage to show all bounties

https://github.com/stakwork/sphinx-tribes/assets/44719946/826c63be-86c0-4c43-90db-5707006d23ac

## Issue ticket number and link
#908 
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by Marvel-Ib** [comment]
@ecurrencyhodler

**Comment by ecurrencyhodler** [comment]
LGTM! Let's do a code review. We will need to test again in staging before we pay out. Thanks marvel.

File: frontend/app/src/people/widgetViews/WidgetSwitchViewer.tsx [hunk 1]
```diff
@@ -103,7 +113,7 @@ function WidgetSwitchViewer(props: any) {
     offer: peopleOffers
   };
 
-  const activeList = [...listSource[selectedWidget]].filter(({ body }: any) => {
```
Comment (kevkevinpal) [inline frontend/app/src/people/widgetViews/WidgetSwitchViewer.tsx:106]: is `listsource` being used anymore? if not we might want to remove it from this file

File: frontend/app/src/store/main.ts [hunk 2]
```diff
@@ -777,7 +777,7 @@ export class MainStore {
     // if we don't pass the params, we should use previous params for invalidate query
     const query2 = this.appendQueryParams(
       'gobounties/all',
-      queryLimit,
```
Comment (kevkevinpal) [inline frontend/app/src/store/main.ts:780]: a simpler change would to be to just increase this queryLimit

File: frontend/app/src/people/widgetViews/WidgetSwitchViewer.tsx (hunk 1 above)
Comment (Marvel-Ib) [reply frontend/app/src/people/widgetViews/WidgetSwitchViewer.tsx:106]: removed

File: frontend/app/src/store/main.ts (hunk 2 above)
Comment (Marvel-Ib) [reply frontend/app/src/store/main.ts:780]: yeah i thought of doing that. but I assumed there was a reason that it was made 100 initially. So i went with the route of if a query limit is passed that should be used and if not it default to the 100 @kevkevinpal

Comment (Marvel-Ib) [reply frontend/app/src/store/main.ts:780]: @kevkevinpal  still waiting for a review

**Comment by elraphty** [comment]
@Marvel-Ib  your changes will break the bounty modal, fix this. 
[Video](https://streamable.com/e/4ebvw2)

**Comment by ecurrencyhodler** [comment]
@Marvel-Ib it looks like raph tested this by going to an old bounty 466. When he did, the website became unresponsive. That is a breaking change. Do you think you can address it?

**Comment by ecurrencyhodler** [comment]
@Marvel-Ib So right now the code is filtering through an array. The load more button calls the backend but the filter only searches through an array. 

There will also need to be refactoring due to new endpoints we have. We should just call the "get bounty" endpoint. 

If you have any other questions, ping @elraphty

**Pull request closed without merging**